bpf/restrictedsession/restricted.bpf.c

```diff
@@ -0,0 +1,279 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+#include <linux/errno.h>
+
+#include "../vmlinux.h"
+#include <bpf/bpf_helpers.h>       /* most used helpers: SEC, __always_inline, etc */
+#include <bpf/bpf_core_read.h>     /* for BPF CO-RE helpers */
+#include <bpf/bpf_tracing.h>       /* for getting kprobe arguments */
+
+#include "../helpers.h"
+
+#define AF_INET     2   /* IP protocol family.  */
+#define AF_INET6    10  /* IP version 6.  */
+
+char LICENSE[] SEC("license") = "Dual BSD/GPL";
+
+// Keeps the set of cgroups which are enforced
+BPF_HASH(restricted_cgroups, u64, u8, 8192);
```

Comment:
Can you update `BPF_HASH`, `BPF_LPM_TRIE`, and `BPF_RING_BUF` sizes to be constants?